app/controllers/assignments_controller.rb

```diff
@@ -9,9 +9,16 @@ class AssignmentsController < ApplicationController
   before_action :set_filter_options, only: %i[show]
   before_action :set_unlinked_users, only: %i[show]
 
+  # rubocop:disable Style/AndOr
   def new
+    if @organization.archived?
```

Comment:
Should we keep this redirect now that there's a model validation that'll fail if they try to use the form?